src/pages/about.js

```diff
@@ -48,9 +49,52 @@ module.exports = React.createClass({
         </span>
       );
     }
+
     this.setState({
       aboutCopy: aboutCopy
     });
+
+    var percentIncrement = 10; // We'll track the video progress in 10% increments.
+    if (this.props.test === "video") {
+
+      this.setState({
+        nextPercentDone: percentIncrement,
+        percentIncrement : percentIncrement,
+        videoStarted : false
+      });
+
+      this.refs.videoPlayer.addEventListener("play", this.onVideoStart);
+      this.refs.videoPlayer.addEventListener("timeupdate", this.onVideoTimeupdate);
+    }
+  },
+  onVideoTimeupdate: function(e) {
+    var video = e.target;
+    var nextPercentDone = this.state.nextPercentDone;
+    var videoProgress = video.currentTime / video.duration * 100;
+
+    if (videoProgress >= nextPercentDone) {
+      this.setState({
+        nextPercentDone: nextPercentDone + this.state.percentIncrement
+      });
+      reactGA.event({
+        category: "Video",
+        action: "Video Progress",
+        label: "Donate Video",
+        value: nextPercentDone
+      });
+    }
+  },
+  onVideoStart: function(e) {
+    if (this.state.videoStarted === false) {
```

Comment:
Instead of having a state variable, you can remove the onVideoStart event on play.

`this.refs.videoPlayer.removeEventListener("play", this.onVideoStart);`

Reply:
@flukeout not sure if you missed this, or just didn't think it would be worth changing. Both are valid reasons :)

If you think it's not worth it, I'm totally fine with keeping it as a state variable.

Reply:
Noticed, but opting to keep how it is as a state variable, although I like your solution more.

Reply:
K, that's fine.